package.json

```diff
@@ -66,9 +66,14 @@
     "build": "react-scripts build",
     "test": "jest",
     "eject": "react-scripts eject",
-    "fix": "eslint --fix --ext js,jsx . && prettier -w .",
-    "lint": "eslint --ext js,jsx --max-warnings 0 . && prettier -c .",
-    "prepare-husky": "husky"
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "lint": "eslint . --ext .js,.jsx,.ts,.tsx",
+    "lint:fix": "npm run lint --fix && npm run format",
```

Comment:
I think I understand where you're going with this code, but we use the linting code in our CI/CD structure, and at that point, we run `npm run lint`. As such, with these changes, the CI/CD won't capture any of the prettier run. I would either revert the lint scripts and lint-related changes or modify `.github/workflows/pr.yaml`.

Reply:
If you could give a little bit more insight into what you were shooting for with these script changes and the `.prettierrc.json` code, I think we could ensure the changes meet the needs of the repo.